backend/src/controllers/access/access.ts

```diff
@@ -62,26 +64,27 @@ export const updateJournal = async (req: Request, res: Response, next: NextFunct
 /**
  * Get the user associated with a journal.
  */
-export const getAccount = async (req: Request, res: Response, next: NextFunction) => {
+export const getAccount = async (
+  req: Request,
+  res: Response,
+  next: NextFunction
+) => {
   const { journalId } = req.params;
 
   try {
-    const journal = await Journal.findById(journalId);
-    if (!journal) return next(new ExpressError('Journal not found.', 404));
-
-    const user = await User.findById(journal.user);
-    if (!user) return next(new ExpressError('User not found.', 404));
-
-    // Journal config is optional
-    const config = await Config.findById(journal.config);
+    const {
+      setConfigMessage,
+      user,
+      config
+    } = await AccountServices.getAccount(journalId);
 
     // If the config doesn't exist instruct the user to set it up
-    if (!config?.model.analysis || !config?.model.chat)
-      req.flash('info', 'Click the Config tab to complete your journal setup.');
-
+    if (setConfigMessage) {
+      req.flash('info', setConfigMessage);
```

Comment:
```suggestion
    if (configMessage) {
      req.flash('info', configMessage);
```

Reply:
I think I see how you were trying to use the name `setConfigMessage` here when passing it to `req.flash`. However, `configMessage` is a more fitting name in this context since it better reflects the value being checked and passed. Using `setConfigMessage` suggests a function or an action that changes state, which isn’t happening here.